Topics/Development_Process/Code Smells/Code_Smells.md

```diff
@@ -0,0 +1,180 @@
+# Code Smells
+
+## Table of Contents
+1. [What is code smell?](#what-is-code-smell)
+2. [Duplicate Code](#duplicate-code)
+3. [Improper Names](#improper-names)
+4. [Dead Code](#dead-code)
+5. [Middle Man](#middle-man)
+6. [Feature Envy](#feature-envy)
+7. [Long Functions](#long-functions)
+8. [Data Clumps](#data-clumps)
+9. [Additional Resources](#additional-resources)
+
+### What is a code smell?
+
+Code smells are indicators of potential issues or weaknesses in software code. They are not bugs or errors, but rather patterns or 
+
+practices that may lead to problems in the future. 
+
+# Common Types of Code Smells and How They Can Harm your Code
```

Comment:
The typesettling hierarchy here seems confusing, maybe change the title here to a ## title, and ### for all the titles within.